.cursor/rules/styles.mdc

```diff
@@ -0,0 +1,14 @@
+---
+description: Rules for consistent styling in component CSS
+globs: *.css
+alwaysApply: true
+---
+
+1. Follow rules outlined in the `../../STYLING.md` file.
```

Comment:
Do we have a `STYLING.md` file? Am I missing it somewhere? Is a guide in the storybook directory? 👀

Reply:
Oh haha no! I think that's actually an SWC thing. I should find a CSS-comparable document.